source/features/remove-zero-packages.tsx

```diff
@@ -0,0 +1,24 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import {getRepoURL} from '../libs/utils';
+
+function init(): void {
+	const packageElem: HTMLElement | null = select(`.numbers-summary a[href='/${getRepoURL()}/packages']`);
+	if (packageElem && packageElem.innerText.trim().startsWith('0')) {
```

Comment:
Are we able to compare here instead of starts with?

Reply:
don't get you. can you please explain further